lib/Game.js

```diff
@@ -1,9 +1,10 @@
 class Game {
-  constructor(player1, player2, player3, round = 1) {
+  constructor(players, round = 1) {
     this.players = [];
     this.currentRound = round;
```

Comment:
Will you ever be initializing a new game that starts at a round other than 1? Seems unecessary to pass it through as a parameter if a new game would imply we're starting over from the beginning.